src/pyats/contrib/plugins/topoup_plugin/README.md

```diff
@@ -0,0 +1,14 @@
+Topology Up Plugin
+------------------
+The topology Up plugin intents to check all the devices connectivity before
+starting the script. It is very useful in the case of virtual devices where
+it can be announced that the device is up and running while the device is actually
+still in the bringup phase.
+
+Arguments:
+```
+TopologyUpPlugin:
+  --all_devices_up                  Enabling/Disabling the plugin
```

Comment:
--all-devices-up

should be - not _

(same for all arguments)

Reply:
Done